src/components/TodoList.js

```diff
@@ -1,15 +1,21 @@
 import List from '@mui/material/List';
-import { TransitionGroup } from 'react-transition-group';
 import {TodoItem} from './TodoItem';
+import {Alert} from "@mui/material";
+import useTodoStore from "../stores/useTodoStore";
 
 export function TodoList() {
-  return (
-    <List sx={{ width: '100%', maxWidth: 360, bgcolor: 'background.paper' }}>
-      <TransitionGroup>
-        {/* render todos here, like this: */}
-        <TodoItem key="1" text="this is an example" completed/>
-        <TodoItem key="2" text="this is 2nd example" completed={false}/>
-      </TransitionGroup>
-    </List>
-  );
-}
\ No newline at end of file
+
+    const todos = useTodoStore((state) => state.list);
+
+    return (
+        <List sx={{width: '100%', maxWidth: 360, bgcolor: 'background.paper'}}>
+            {
+                todos.length > 0
+                    ? todos.map((todo, index) =>
+                        <TodoItem key={index} index={index} text={todo.text} completed={todo.completed}/>
+                    )
+                    : <Alert severity="info">There are no todos to show.</Alert>
```

Comment:
it would be better to set the Alret component outside of the List component because the List component(represent ul HTML tag) expects to have a ListItem component(represent li HTML tag) so it can be like this

```
    if(todos.length === 0) return (<Alert severity="info">There are no todos to show.</Alert>);
    
    return (
        <List sx={{width: '100%', maxWidth: 360, bgcolor: 'background.paper'}}>
            {todos.map((todo, index) =>
                <TodoItem key={index} index={index} text={todo.text} completed={todo.completed}/>
            )}
        </List>
    );
```